src/Listeners/NotifyOnBounce.php

```diff
@@ -2,23 +2,34 @@
 
 namespace Vormkracht10\Mails\Listeners;
 
-use Illuminate\Support\Facades\Notification;
 use Vormkracht10\Mails\Events\MailBounced;
+use Vormkracht10\Mails\Listeners\Concerns\SendsNotifications;
 use Vormkracht10\Mails\Notifications\BounceNotification;
 
 class NotifyOnBounce
 {
+    use SendsNotifications;
+
     public function handle(MailBounced $event): void
     {
-        if (null !== $notificationChannels = config('mails.notifications.events.bounce.notify')) {
-            collect($notificationChannels)->each(function ($channel) use ($event) {
-                $accounts = config('mails.notifications.channels.'.$channel);
-
-                collect(array_wrap($accounts))->each(function ($account, $route) use ($channel, $event) {
-                    Notification::route($channel, $account)
-                        ->notify(new BounceNotification($event->mailEvent->mail));
-                });
-            });
+        if (! $channels = config('mails.events.bounce.notify')) {
+            return;
+        }
+
+        foreach ($channels as $channel) {
```

Comment:
Dit stukje is duplicated code op meerdere plekken, kun je hier iets doen met die SendsNotifications trait waar je alleen aan doorgeeft de variabelen en de notificatie die verzonden moet worden?